src/main/java/com/github/awwkoala/selenium/page/CheckboxesPage.java

```diff
@@ -0,0 +1,59 @@
+package com.github.awwkoala.selenium.page;
+
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class CheckboxesPage extends Page {
+
+  @FindBy(css = "input[type='checkbox']")
+  private List<WebElement> checkboxes;
+
+  public CheckboxesPage(WebDriver driver) {
+    super(driver);
+  }
+
+  public void goToCheckboxesPage() {
+    driver.get("http://theinternet.przyklady.javastart.pl/checkboxes");
+  }
+
+  public void clickCheckbox1() {
+    checkboxes.get(0).click();
+  }
+
+  public void clickCheckbox2() {
+    checkboxes.get(1).click();
```

Comment:
Did you think about fact that Page could not loaded yet? What about that case Selenium will have empty `checkboxes` list and you're trying to call second element from that

Reply:
You could think about some wait mechanism to waitUntilIsVisilble/ clickable and then click.